engine/runtime/src/main/java/org/enso/interpreter/node/callable/argument/ReadArgumentNode.java

```diff
@@ -73,6 +73,6 @@ public Object executeGeneric(VirtualFrame frame) {
 
   @Override
   public boolean isInstrumentable() {
-    return false;
+    return true;
```

Comment:
Obviously the amount of _instrumentable nodes_ goes up. Hence the
- `Execution creates some nodes. Maximal size should be 40` failures
- at various [CI checks](https://github.com/enso-org/enso/actions/runs/14638260720/job/41074416934?pr=12936#step:10:5144)

It is fine to adjust the expected number of instrumented nodes in the test.